ui/shared/components/panel/family.jsx

```diff
@@ -58,7 +58,7 @@ BaseFirstSample.propTypes = {
 }
 
 const mapSampleDispatchToProps = (state, ownProps) => ({
-  firstFamilySample: getFirstSampleByFamily(state)[ownProps.familyGuid],
+  firstFamilySample: (getSamplesByFamily(state)[ownProps.familyGuid] || [])[0],
```

Comment:
`getFirstSampleByFamily` didn't do anything other than this one line, so storing essentially a copy of the data in memory wasn't worth it